src/SFA.DAS.ApplyService.Web/Views/Application/Pages/Index.cshtml

```diff
@@ -83,7 +83,8 @@
                     </div>
                 }
 
-                <form method="post" asp-action="SaveAnswers" asp-controller="Application" novalidate enctype="multipart/form-data">
+                @* no action is specified for the form, post back to the get url to preserve the page hierarchy parameters *@
+                <form method="post" novalidate enctype="multipart/form-data">
```

Comment:
And this one.

Reply:
This is necessary comment as what this does is unexpected, post back to get url is a non-typical form post style